core/src/main/java/org/apache/iceberg/rest/HTTPClient.java

```diff
@@ -96,6 +99,9 @@ private HTTPClient(
       clientBuilder.addRequestInterceptorLast(requestInterceptor);
     }
 
+    int maxRetries = PropertyUtil.propertyAsInt(properties, REST_MAX_RETRIES, 5);
```

Comment:
This is a higher retry count than before with the default (which is 1 retry I believe), just wanted to double check that was the intent.

Reply:
yes this was the intent to increase the retry